src/stacks/angular/index.ts

```diff
@@ -4,19 +4,13 @@ import { StackRegister } from '../stack-register';
 export class Angular {
   constructor(private readonly rootPath: string = './') {}
 
-  // isAvailable() {
-  //   try {
-  //     const packageJson = require(`${this.rootPath}/package.json`);
-  //     return Object.keys(packageJson.dependencies).includes('@angular/core');
-  //   } catch (e) {
-  //     return false;
-  //   }
-  // }
-
   async isAvailable(): Promise<boolean> {
     try {
       const packageJSON = require(`${this.rootPath}package.json`);
-      return Object.keys(packageJSON.dependencies).includes('@angular/core');
+      if (packageJSON.dependencies !== undefined) {
```

Comment:
We should maybe create an utility method in order to check if the project as something as a dep or devDep